### DISCUSSION TITLE
Button for linking to a more info / instruction page

### DISCUSSION THREAD

**Discussion by AvalancheAddict** [chuot/rdio-scanner#102] [Ideas]
I have what I think would be a relatively easy to implement and very helpful idea.

I think a simple Button on either the main screen (like to the left of the Settings button) or at least somewhere in the Select TG panel that would allow for a simple web link to be defined by the system administrator to take them to an external website for more information.  On an iPhone, the link could bring up Safari while leaving the Rdio-Scanner app still running.  On a PC, you would probably want it to open a new tab.  I don't have an Android, but I assume handling it the same way as the iPhone would be viable.

For instance, I could use any sort of free hosting site (Blogger for example) and I could then create a page or series of pages that would cover things like the following:

- How to use this specific system
- What frequencies / channels / services does the system cover
- Explaining what certain abbreviations on the buttons mean
- Department Listings (some areas use all numbers for the Fire Department dispatches, so you need an easy way look them up to know where they are)
- Unit Number Listings
- Radio Codes and Jargon
- Other useful info, like Highway Exit Numbers & Mile Markers, local maps, etc.  Anything that a listener might want to refer to while monitoring.
- Contact info for the system owner if so desired

Thanks for your consideration.  Maybe it's not as easy to implement as I think, but for me this would be a killer feature and hopefully others feel the same way.

**Comment by chuot** [comment]
You know that the native app have some sort of a built-in help. Just long-press any of the buttons to get contextual help.

Thanks for your idea.

**Reply by AvalancheAddict** [reply]
Yes.  I was referring more to user instructions specific to the system itself and how the system admin has things laid out, and not the functionality of the App, as that is fairly intuitive.

Actually, I don't know why I didn't think of it before, but if you don't want to have it fire off a link to a external site, even a button that brings up a text box just like you get when you click on "Terms of Use" and "Privacy Policy", but instead have the text in the box be defined by the system admin, it would provide a simple space for an admin to put at least some of the info I mentioned into the App..

Thanks for your consideration.

**Comment by blackwellj** [comment]
This would be a nice feature. Especially being able to link to other pages
